venvUtils/ensureVenv.py

```diff
@@ -94,7 +94,7 @@ def populate():
 			"-m",
 			"pip",
 			"install",
-			"setuptools==69.5.1",
+			"setuptools==70",
```

Comment:
setuptools is also mentioned in pyproject.toml, please ensure this is updated to reference the latest setuptools.

The latest setuptools is 72, have you tested this version? if so, what didn't work.

Please fixup the comment above which references py2exe and setuptools